charts/cadc-tap/values.yaml

```diff
@@ -212,6 +213,51 @@ uws:
   # -- Affinity rules for the UWS database pod
   affinity: {}
 
+cloudsql:
+  # -- Enable the Cloud SQL Auth Proxy sidecar, used with Cloud SQL databases
+  # on Google Cloud
+  enabled: false
+
+  image:
+    # -- Cloud SQL Auth Proxy image to use
+    repository: "gcr.io/cloudsql-docker/gce-proxy"
+
+    # -- Cloud SQL Auth Proxy tag to use
+    tag: "1.35.3"
+
+    # -- Pull policy for Cloud SQL Auth Proxy images
+    pullPolicy: "IfNotPresent"
+
+  # -- Instance connection name for a Cloud SQL PostgreSQL instance
+  instanceConnectionName: ""
+
+  # -- The Google service account that has an IAM binding to the `vo-cutouts`
+  # Kubernetes service accounts and has the `cloudsql.client` role, access
+  # to the GCS bucket, and ability to sign URLs as itself
+  # @default -- None, must be set
+  serviceAccount: ""
+
+  # -- Resource limits and requests for the Cloud SQL Proxy container
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "100m"
+      memory: "20Mi"
+    requests:
+      cpu: "5m"
+      memory: "7Mi"
+
+serviceAccount:
+  # -- Specifies whether a service account should be created.
+  create: false
+
+  # -- Annotations to add to the service account
+  annotations: {}
+
+  # The name of the service account to use.
+  # @default -- Generated using the fullname template
+  name: "cadc-tap"
```

Comment:
It looks like this `@default` note is incorrect; I'm guessing you tried doing that and it didn't work? It would be great if we could have it pick up the deployment name that way, but passing in the service account name explicitly is fine. I would set this to null and say something like `@default -- None, must be set if service account creation is enabled`, though, as noted above.

Reply:
For this one there may be a better solution that I was missing. Basically I wanted the serviceaccount template which is in the cadc-tap chart to end up with the name of the application that is using the cadc-tap as a dependency, e.g. "ssotap" for the ssotap application. Didn't find a better solution for this, so ended up just having it passed in as a name value like this. (The default note is indeed incorrect and I've changed that)